src/components/UpcomingEvents/index.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import './styles.css';
+import { charityAPI } from '../../clients';
+// Heading global
+import Heading from '../Heading'
+// Date Package
+import {parseISO} from 'date-fns';
+// Images
+import bg_1 from './img/bg_1.png';
+
+// Some Other Components
+export const BackgroundImage = () => (
+  <div className="upcoming-events__bg-image absolute w-2/5">
+    <img src={bg_1} alt="upcoming events Background" />
+  </div>
+);
+
+const Events = props => {
+// Function to get add dates needed
+function getDate(myDate) {
```

Comment:
I ADMIRE how you 'UNBLOCKED' yourself and applied the **RESULT-ORIENTED** concept perfectly in this function. 
For other team members, This is a good example/approach to follow when you don't know how to solve a problem, do not stick for a longer time, deliver the desired task and improve it iteratively (later)

however, this doesn't prevent me from suggesting the change 
```js
function getDate(myDate) {
    const theDate = parseISO(myDate);

    return{
      month: format(theDate, 'MMM'),
      day: format(theDate, 'dd'),
      year: format(theDate, 'yyyy'),
      time: format(theDate, 'hh:mm 	a').toLowerCase()
    };
  }
```
Very good job ya nag3awy!👍🏼

Reply:
Thanks I'm going to look to it